server/middleware/populateCurrentUser.ts

```diff
@@ -72,10 +72,11 @@ export default function populateCurrentUser(userService: UserService, licenceSer
             cvlUser.telephoneNumber = probationUser?.telephoneNumber
             cvlUser.probationAreaCode = probationUser?.probationArea?.code
             cvlUser.probationAreaDescription = probationUser?.probationArea?.description
-            cvlUser.probationPduCodes = probationUser?.teams?.map(team => team?.borough?.code)
-            cvlUser.probationLauCodes = probationUser?.teams?.map(team => team?.district?.code)
-            cvlUser.probationTeamCodes = probationUser?.teams?.map(team => team?.code)
-            cvlUser.probationTeams = probationUser?.teams
+            const teams = probationUser?.teams?.filter(t => !t.endDate || parseIsoDate(t.endDate) > new Date()) || []
+            cvlUser.probationPduCodes = Array.from(new Set(teams.map(team => team?.borough?.code)))
+            cvlUser.probationLauCodes = Array.from(new Set(teams.map(team => team?.district?.code)))
```

Comment:
prevent duplicates appearing in the list when someone exists on multiple teams in the same LAU/PDU